csrc/tensor_metadata.cpp

```diff
@@ -81,17 +81,14 @@ class ForwardTraverseFromLogicalToAlloc {
     auto [outer_size, outer_stride] = outer_it->second;
     NVF_ERROR(
         inner_stride * inner_size == outer_stride,
-        "The logical domain and allocation domain of fusion input/output ",
-        "tensors must be a one-to-one map, therefore, ",
```

Comment:
I updated this comment -- we support beyond one-to-one but merging of discontiguous dimensions is still disallowed for a different reason.